templates/results-row.php

```diff
@@ -1,17 +1,17 @@
 <tr class="plugin-check__results-row">
-	<td>
+	<td data-label="<?php esc_html_e( 'Line', 'plugin-check' ); ?>">
```

Comment:
Why not `esc_attr` since it's within an attribute?

Reply:
ok